platform-ui-skin/pom.xml

```diff
@@ -37,6 +37,7 @@
             <plugin>
                 <groupId>org.lesscss</groupId>
                 <artifactId>lesscss-maven-plugin</artifactId>
+                <version>1.7.0.1.1</version>

```

Comment:
-> Do you really need to upgrade the version of this plugin for your fix ?

No